docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/singularity.rst

```diff
@@ -126,3 +128,55 @@ cache:
             temporary-image \
             docker://$image
       scp temporary-image mycluster:$singularity_image_root/$image
+
+.. _manage-singularity-cache:
+
+********************************************************************************
+ Managing the Singularity Image Cache using the manage-singularity-cache script
+********************************************************************************
+
+A convenience script, ``/etc/launcher/scripts/manage-singularity-cache``,is provided by the HPC
+launcher installation to simplfiy the management of the Singularity image cache. The script
+simplifies the management of the Singularity image cache directory content and helps ensure proper
+name, placement, and permissions of content added to the cache. Adding container images to the
+Singularity image cache avoids the overhead of downloading of the images and allows for sharing of
+images between multiple users. It provides the following features:
+
+   -  Download the Determined default cuda, cpu, or rocm environment images
+   -  Download an arbitrary docker image reference
+   -  Copy a local Singularity image file into the cache
+   -  List the currently available images in the cache
+
+If your system has internet access, you can download images directly into the cache. Use the
+``--cuda``, ``--cpu``, or ``--rocm`` options to download the current default CUDA, CPU, or ROCM
+environment container image into the cache. For example, to load the default CUDA container image,
+use the following command:
+
+   .. code:: bash
+
+      /etc/launcher/scripts/manage-singularity-cache --cuda
+
+If your system has internet access, you can download any desired docker container image (e.g.
+``determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730``) into the cache using the
+command:
+
+   .. code:: bash
+
+      /etc/launcher/scripts/manage-singularity-cache determinedai/environments:py-3.8-pytorch-1.10-tf-2.8-cpu-096d730
+
+Otherwise, from an internet-connected system, download the desired image using the Singularity pull
```

Comment:
Does "Singularity pull" need to be enclosed in double back quotes and made lowercase?  That is, if your intent is to reference "singularity pull".

Reply:
Intent was not to specify the actual syntax (which is in the example).   I did change the command to ``pull`` though as that is the literal comman.